### DISCUSSION TITLE
Model below build plate

### DISCUSSION THREAD

**Discussion by AngryMammoth** [OrcaSlicer/OrcaSlicer#339] [Ideas]
Any chance we can get the ability to drop the model below the build plate like you can in cura? By either the arrows or -Z value. I know we can use the "cut" feature, but being able to just insert -.15 Z would be very nice so we can control and adjust on the fly.

But thank you for all your hard work. Please know it's appreciated.

**Comment by Padawg** [comment]
I also miss this feature after years of using it in CURA. Hopefully it will be implemented.
